### ISSUE TITLE
Not-compatible out-of-the-box with datasets (low-fidelity or 0.5 bitwise accuracy).

### ISSUE THREAD

**Issue by sebszyller** [ningyu1991/ArtificialGANFingerprints#2]
Hi, I'm trying to produce some fingerprints using your code.
I've been experimenting with AFHQ (cats only) but to no avail.

First of all, the bitwise accuracy stays at 50% for most runs and occasionally it converges to sth like 90%+. But when it does converge, the fidelity of the output images is virtually nonexistent.

I'm running the autoencoder training as `python3 train.py --data_dir data/afhq/train/cat/ --image_resolution 128 --output_dir ./output_cat --fingerprint_length 100 --batch_size 32 --num_epochs 30`.

Followed by `python3 embed_fingerprints.py --encoder_path output_cat/checkpoints/stegastamp_100_28102021_10:00:00_encoder.pth --data_dir data/afhq/train/cat --image_resolution 128 --output_dir output_cat_encoded --batch_size 16`.

Am I missing something?
